sktime/forecasting/ensemble_algorithms.py

```diff
@@ -0,0 +1,234 @@
+from sklearn.base import BaseEstimator
+from scipy.optimize import nnls
+import numpy as np
+
+
+class EnsembleAlgorithm(BaseEstimator):
+    """Wrapper class to take ensemble algorithms and allow them to be easily set
+
+    Parameters
```

Comment:
in docstrings of functions and classes, for arguments and parameters, please always say:
* name, type, explanation, e.g., `n: float - number of estimators`
* expected structure, if a specific one is required, e.g., `loss_func: loss function, following the `sklearn.metrics` API - loss function to used in updating of the weights`

In docstrings of classes, please try to give:
* a description of behaviour (what it does/implements)

Reply:
I tried to update to follow this better, let me know if any are still not named properly.